expan/core/statistics.py

```diff
@@ -57,7 +58,8 @@ def delta(x, y, assume_normal=True, percentiles=[2.5, 97.5],
             the weighted mean and confidence intervals, which is equivalent
             to the overall metric. This weighted approach is only relevant
             for ratios.
-        num_tests: number of tests or reported kpis
+        multi_test_correction (boolean): correct the confidence intervals (multiple correction problem)
+        num_tests: number of tests or reported kpis.
```

Comment:
how about "num_tests (integer): number of tests or reported kpis used for multiple correction. This value is only used if multi_test_correction=True"?